Tasks/src/Calculator.java

```diff
@@ -0,0 +1,35 @@
+import java.util.Scanner;
+
+public class Calculator {
+
+        public static void main (String[] args) {
+            double sum, num1, num2;
+            Scanner scd = new Scanner(System.in);
+            Scanner scs = new Scanner(System.in);
+            System.out.println("Давай попробуем что-то посчитать");
+            System.out.println("Введи первую цифру...");
+            num1 = scd.nextDouble();
+            System.out.print(num1);
+            System.out.println(" |Теперь оператор...");
+            String arg = scs.nextLine();
+            while (!arg.equals("+")&!arg.equals("-")&!arg.equals("*")&!arg.equals("/")&!arg.equals("%")) {
```

Comment:
Please make it handle all String or remove

Reply:
Did not understand the comment. Can you describe the problem in more detail?